workspaces/kener/plugins/kener-status/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint.backend')],
+  rules: {
+    'notice/notice': 'off',
```

Comment:
This rules are intended to be there across the entire repo, please do not turn them off. This is part of contributing a plugin to this repo.